libs/telemetry/src/main/java/org/opensearch/telemetry/tracing/DefaultTracer.java

```diff
@@ -24,7 +24,7 @@
  *  @opensearch.internal
  */
 @InternalApi
-class DefaultTracer implements Tracer {
+public class DefaultTracer implements Tracer {
```

Comment:
Please revert, there are no reasons to expose the implementation class, please use `Tracer` APIs